### DISCUSSION TITLE
Whats is the requirment to run Monika after story mod

### DISCUSSION THREAD

**Discussion by Glitch2545** [Monika-After-Story/MonikaModDev#10029] [QA - tech support + general] [answered]
What verison of DirectX you need to run the mod and openGL verison too please

**Comment by Booplicate** [comment]
Just use the latest DirectX available for your system.

**Comment by Glitch2545** [comment]
> Just use the latest DirectX available for your system.

I have directX 12 but the options is invisible like this https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#all-the-buttons-look-like-this Please  I tried all the solutions responde fast

**Comment by ThePotatoGuy** [comment]
#9937 

have you tried dx11 then? since MAS runs on an older ver of renpy it might not be compatible with dx12. 

otherwise there is nothing more we can do. we've ruled out win11, graphics drivers, and you say you can run renpy.

**Comment by Booplicate** [comment, marked as answer]
dx12 should be fine. Looking at the other issue, I'd rather blame the integrated GPU. But also at least one of options from [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#all-the-buttons-look-like-this) would work (even if in 15 FPS), so that means they didn't correctly configure it.
